server/webhook.go

```diff
@@ -163,6 +163,23 @@ func (p *Plugin) permissionToRepo(userID string, ownerAndRepo string) bool {
 	return true
 }
 
+func (p *Plugin) excludeConfigOrgMember(user *github.User, subscription *Subscription) bool {
+	if !subscription.ExcludeOrgMembers() {
+		return false
+	}
+
+	info, err := p.getGitHubUserInfo(subscription.CreatorID)
+
+	if err != nil {
+		return false
```

Comment:
We should log the error here before returning possibly as a warning

Reply:
Done, thanks!